docs/shared/migration/adding-to-existing-project.md

```diff
@@ -322,19 +322,35 @@ To connect to Nx Cloud:
 - Commit and push your changes
 - Go to [https://cloud.nx.app](https://cloud.nx.app), create an account, and connect your repository
 
-#### Connect to Nx Cloud Manually
+```shell
+npx nx connect
+```
+
+Follow the steps and make sure Nx Cloud is enabled on the main branch.
+
+### Generate a CI Workflow
```

Comment:
This section is already present further up as well.

tbh we should just need to run `nx connect` which also asks about generating a CI setup